src/test/java/com/google/api/codegen/testdata/php_client_impl_library.baseline

```diff
@@ -645,7 +570,7 @@ class LibraryServiceGapicClient
      * ```
      * try {
      *     $libraryServiceClient = new LibraryServiceClient();
-     *     $formattedName = LibraryServiceClient::formatShelfName("[SHELF_ID]");
+     *     $formattedName = $this->shelfPath("[SHELF_ID]");
```

Comment:
Why "path" instead of "name"? These represent resource names. While they have the form of a path, that doesn't mean that they can ever be used as an actual path as a subcomponent of a server URL.

Reply:
@bshaffer @dwsupplee @jdpedrie @vam-google Gentle ping, any thoughts?

Reply:
I agree with @garrettjonesgoogle, especially after taking a look at the documentation for services such as spanner and pubsub. The resource seems to always be referred to as "names", so it makes sense to me to match that here as well.

Reply:
Updated to `*Name`